packages/flutter/lib/src/cupertino/default_text_selection_toolbar.dart

```diff
@@ -0,0 +1,205 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/foundation.dart' show defaultTargetPlatform;
+import 'package:flutter/widgets.dart';
+
+import 'desktop_text_selection_toolbar.dart';
+import 'text_selection_toolbar.dart';
+import 'text_selection_toolbar_buttons_builder.dart';
+
+/// The default Cupertino context menu for text selection for the current
+/// platform.
+///
+/// Builds the mobile Cupertino context menu for all mobile platforms, not just
+/// iOS, and builds the desktop Cupertino context menu for all desktop
+/// platforms, not just MacOS. For a widget that builds the native-looking
+/// context menu for all platforms, see [DefaultTextSelectionToolbar].
+///
+/// The children can be customized using the [children] or [buttonItems]
+/// parameters. If neither is given, then the default buttons will be used.
+///
+/// See also:
+///
+/// * [EditableTextContextMenuButtonItemsBuilder], which generates the default
+///   [ContextMenuButtonItem]s for the current platform for a context menu
+///   displaying inside of an [EditableText].
+/// * [TextSelectionToolbarButtonsBuilder], which builds the native-looking
+///   button Widgets for the current platform given [ContextMenuButtonItem]s.
+/// * [DefaultTextSelectionToolbar], which does the same thing as this widget
```

Comment:
This seems to contradict what it says above.

Reply:
I reworded it, thanks.